tests/eko/test_ev_operator.py

```diff
@@ -30,7 +30,8 @@ def test_quad_ker(monkeypatch):
         res_ns = quad_ker(
             u=0,
             order=0,
-            mode="NS_p",
+            mode0=br.non_singlet_pids_map["ns+"],
```

Comment:
Since all occurrences of these `br.non_singlet_pids_map[...]` are quite verbose for a number, I wonder if we should provide even further aliases in `br`, like `br.nsp` and related.

Reply:
Maybe we can do (or not do) this later ...